eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/ClusteredVariant.java

```diff
@@ -44,6 +44,7 @@
     public ClusteredVariant(IClusteredVariant variant) {
         this(variant.getAssemblyAccession(), variant.getTaxonomyAccession(), variant.getContig(), variant.getStart(),
              variant.getType(), variant.isValidated());
+        this.createdDate = variant.getCreatedDate();
```

Comment:
I think it could be a more intuitive interface if we added the created date to the list of parameter in the other constructor. The default will be `null` many times, but all the constructors will behave the same way, instead of one setting the date and the other don't.